local/client/cli/echo_cmd.go

```diff
@@ -25,6 +24,6 @@ func (c *CliClient) echo(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	log.Println("Scoot server: ", result)
+	fmt.Println(result)
```

Comment:
yes; log includes info like a timestamp (and I think prints to stderr?)

fmt.Print* print straight to stdout, which is what you want for a Unix tool.